src/main/java/net/atos/entng/support/helpers/UserInfosHelper.java

```diff
@@ -87,4 +88,10 @@ public static List<UserInfos.Action> getUserActionsFromJSONArray(JsonArray actio
         return ((List<JsonObject>) actions.getList()).stream().map(UserInfosHelper::getUserActionFromJSON).collect(Collectors.toList());
      }
 
+    public static String getAppName(UserInfos user, String appAddress) {
+        return user.getApps().stream()
+                .filter(app -> Objects.equals(app.getAddress(), appAddress))
+                .map(UserInfos.Application::getDisplayName).findFirst().orElse("");
```

Comment:
Une instruction par ligne
```suggestion
        return user.getApps().stream()
                .filter(app -> Objects.equals(app.getAddress(), appAddress))
                .map(UserInfos.Application::getDisplayName)
                .findFirst()
                .orElse("");
```